daemons/lvmdbusd/manager.py

```diff
@@ -266,3 +266,12 @@ def Uuid(self):
 		"""
 		import uuid
 		return uuid.uuid1()
+
+	@dbus.service.method(
+		dbus_interface=MANAGER_INTERFACE,
+		in_signature='a{sv}')
+	def SetCmdLineOptions(self, options):
```

Comment:
I am little bit afraid this could cause a lot of issues when two different tools both use this function to set their own custom command line options to add.

Reply:
> I am little bit afraid this could cause a lot of issues when two different tools both use this function to set their own custom command line options to add.

Indeed.  With the current implementation, the options replace whatever was present before.  So multiple clients would step on each other.  I'll remove this change from patch set.